app/Http/Controllers/VolunteerApiController.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+use App\Http\Requests;
+
+use App\User;
+
+class VolunteerApiController extends Controller
+{	
+	
+    public function show($id)
+    {
+    	$volunteer = User::findOrFail($id);
+
+    	return response()->toJson(['organizationReviews' => $volunteer->organizationReviews()->get(), 'eventReviews' => $volunteer->eventReviews()->get(), 'followedEvents' => $volunteer->followedEvents()->get(), 'registeredEvents' => $volunteer->registeredEvents()->get()]);
```

Comment:
well, I guess according to mostafa it ended up my way wasn't wrong, it was actually more correct :). https://laravel.com/docs/5.1/responses#json-responses
Anyway, I will add more attributes to the object in case they were needed although they weren't specified in the issue.